larq_compute_engine/mlir/ir/lce_ops.td

```diff
@@ -50,12 +50,15 @@ TODO
     TensorOf<[F32]>:$post_activation_multiplier,
     TensorOf<[F32]>:$post_activation_bias,
 
+    DefaultValuedAttr<I32Attr, "-1">:$channels_in,
```

Comment:
I am not a fan of this argument, but I guess for now this is the easiest way to preserve the shape :+1:

Reply:
Yes at some point we can have some nice bitpacked tensor type that preseves this, but it's the only easy way for now.